WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -118,5 +89,60 @@ protected override void ResetUi()
 			base.ResetUi();
 			PayjoinEndPoint = "";
 		}
+
+		private IPsbtSigner GetSigner()
+		{
+			return !Wallet.KeyManager.IsHardwareWallet
+				? null
+				: new HwiPsbtSigner(new HwiClient(Global.Network), b => IsHardwareBusy = b);
+		}
+
+		private class HwiPsbtSigner : IPsbtSigner
```

Comment:
Maybe this class should be in a separate file?

Reply:
done though I left it in the same directory since I'm not exactly sure where to put it